tests/console/validate_zfcp.pm

```diff
@@ -0,0 +1,202 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate zfcp in the installed system:
+#          - Verification of FCP devices online
+#          - Verificaiton of FCP devices
+#          - Verification of LUNs visible as SCSI devices
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use utils qw(arrays_subset);
+
+# Get devices online.
+# For each physical adapter in the mainframe there is a corresponding FCP channel providing physical
+# connection to the SAN Fabric (multiple FCP channels are configured to increase the I/O bandwidth
+# and improve data availability).
+# HBAs are shown in the system as FCP devices
+sub get_fcp_devices_online {
+    record_info('FCP devices', 'Verification of FCP devices online');
```

Comment:
i am still not sure about this description. you grep the test_data but you dont do any verification against the SUT. seems that this belongs inside the `verify_fcp_devices` but there is already description for it. Do you really need a record_info here?

Reply:
I can name it "Get expected devices online" if helps due to verification is against the test_data (which is a little bit to do the test more smart that it should but here it is not your point).

Reply:
"Get expected devices online" sound right